js/exporter/jspdf/export.js

```diff
@@ -1,4 +1,4 @@
-import { isDefined, isObject, isFunction } from '../../core/utils/type';
+import { isDefined, isObject, isFunction, isNumeric } from '../../core/utils/type';
 import { extend } from '../../core/utils/extend';
 import dateLocalization from '../../localization/date';
 
```

Comment:
Can we add '_' prefix to 'private' functions in this PR?
Or, reduce export to
```
export const Export = {
  export: function(options) { .. } - the only exported function ?
}
```

Reply:
yes, I will add the "_" prefix for all methods except the "export (...)" method.

Reply:
Left without underlining. Like in the code for exporting to Excel.